docs/DeveloperGuide.md

```diff
@@ -965,3 +847,17 @@ testers are expected to do more *exploratory* testing.
    1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
 
 1. _{ more test cases …​ }_
+
+### Resetting all ingredients' levels to zero
```

Comment:
This part should be before the saving data

Reply:
Okay thanks!